pkg/kubelet/stats/cadvisor_stats_provider.go

```diff
@@ -294,9 +294,9 @@ func isPodManagedContainer(cinfo *cadvisorapiv2.ContainerInfo) bool {
 	podNamespace := kubetypes.GetPodNamespace(cinfo.Spec.Labels)
 	managed := podName != "" && podNamespace != ""
 	if !managed && podName != podNamespace {
-		klog.Warningf(
-			"Expect container to have either both podName (%s) and podNamespace (%s) labels, or neither.",
-			podName, podNamespace)
+		klog.InfoS(
+			"Expect container to have either both podName and podNamespace labels, or neither",
+			"podNameLabel", podName, "podNamespaceLabel", podNamespace)
```

Comment:
why not use  klog.KRef instead?

Reply:
I didn't use klog.KRef because this log entry is emitted when kubelet decides that the container is not a part of any pod, but for some reason has one of the two labels